skin-resque-frontend/src/pages/FrontPage/FrontPage.js

```diff
@@ -23,7 +70,21 @@ function FrontPage() {
         </div>
       </div>
       <img src={photo} className='skinPhoto'></img>
-      <div className='white'></div>
+      <div className='white-info'>
```

Comment:
Will this sort of structure used elsewhere? If so you should consider making it standalone component